.github/workflows/crossplane-upgrade.yml

```diff
@@ -33,7 +33,7 @@ jobs:
         run: kubectl create namespace crossplane-system
         shell: bash
       - name: Install Crossplane from Stable
-        run: helm repo add crossplane-stable https://charts.crossplane.io/stable && helm repo update && helm install crossplane --namespace crossplane-system crossplane-stable/crossplane --version 1.1.0 --wait
+        run: helm repo add crossplane-stable https://charts.crossplane.io/stable && helm repo update && helm install crossplane --namespace crossplane-system crossplane-stable/crossplane --version 1.9.0 --wait
```

Comment:
i don't think it's the right thing to do long term to be installing a specific version of Crossplane here (or the new instances you've introduced below), instead of just getting the latest from stable channel.  If we don't specify a version here, then we don't have to keep updating this value over time.

But let's keep it consistent for now and track that with an issue :)

Reply:
So, do I open an issue pertaining to that?

Reply:
@parul5sahoo that would be great!

Reply:
we discussed this in our sync-up this morning, and @hasheddan confirmed that we want to use explicit version numbers in these helm install commands, so we have confidence that the version we're installing from stable channel is the same version of Crossplane tests that we are cloning from the release branch.

@jbw976 will follow up to add a reminder note in the crossplane release issue template for whoever is running the release to update these version numbers.

Reply:
PR to update the crossplane release steps: https://github.com/crossplane/crossplane/pull/3219